### ISSUE TITLE
Player stuck on destroying blocks

### ISSUE THREAD

**Issue by IAbstract** [SirCmpwn/TrueCraft#216]
Installed on Win10. When I spawn in world, my avatar won't stop breaking blocks.

**Comment by SirCmpwn** [comment]
Do you have a gamepad plugged in?

**Comment by IAbstract** [comment]
No. Using a laptop and mouse. Do you think that the touchpad could be problematic?

**Comment by SirCmpwn** [comment]
Perhaps. Can you try clicking your mouse and tapping the touchpad? That is, produce the inputs that would normally start digging to "flush" the system.

**Comment by IAbstract** [comment]
Did all the clicking to no avail.

**Comment by SirCmpwn** [comment]
:(
